src/openforms/forms/api/serializers/form_definition.py

```diff
@@ -161,6 +161,14 @@ def validate(self, attrs):
 
         return attrs
 
+    def save(self, **kwargs):
+        # field 'component_translations' has been deprecated and it's not part of
+        # the 'form_definition' model anymore. We use it only in the serializer.
+        if "component_translations" in self.validated_data:
+            del self.validated_data["component_translations"]
+
+        return super().save(**kwargs)
```

Comment:
as this probably only applies to the import process, I'd move this to `openforms.forms.utils.import_form_data` and drop the data right before `instance = deserialized.save()` is called. There already is some data pre-processing happening in there :)